src/pc/stats.rs

```diff
@@ -194,8 +194,8 @@ impl ParachainStats {
 	}
 
 	/// Update skipped slots count
-	pub fn on_skipped_slot(&mut self) {
-		self.skipped_slots += 1;
+	pub fn on_skipped_slot(&mut self, block_number: u32) {
+		self.skipped_slots.push(block_number);
```

Comment:
If the tool runs for long, this will fill up memory. I think we should have a hard limit on how many of this hashes we keep. Ideally we should drop the oldest ones when we hit the limit to make room for newer hashes.

Reply:
Yes, I was about to suggest using of the `[VecDeque](https://doc.rust-lang.org/std/collections/struct.VecDeque.html#)` instead of the `Vec` here.

Reply:
@sandreim Should the amount be really small, like 10, because we use CLI.

Reply:
It can also be started in Prometheus mode, so if a relay chain stalls, this will have hashes for all relay blocks multiplied by number of parachains.

Reply:
> @sandreim Should the amount be really small, like 10, because we use CLI.

yeah, that makes sense as a default value, but let's add a CLI option to override it.

Reply:
Added